modelmapper/etl.py

```diff
@@ -141,11 +134,15 @@ def _extract(self, session, path=None, content=None, content_type=None,
         if use_client:
             content = self.get_client_data()
 
+        if isinstance(content, tuple):
```

Comment:
Can you please leave a comment why content can be a tuple?

Reply:
👍